sdk/devcenter/developer-devcenter-rest/swagger/README.md

```diff
@@ -0,0 +1,31 @@
+## C
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+```yaml
+package-name: "@azure-rest/developer-devcenter"
+title: "Azure DevCenter"
+description: "Azure DevCenter Client"
+generate-metadata: false
+openapi-type: data-plane
+tag: v2022-03-01-preview
+generate-test: true
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../
+source-code-folder-path: ./src
+package-version: 1.0.0
+rest-level-client: true
+security: AADToken
+security-scopes: https://devcenter.azure.com/.default
+use-extension:
+  "@autorest/typescript": "6.0.0-rc.1"
+
+input-file:
+  -  C:\Users\chrismiller\source\repos\azure-devtest-center\src\sdk\specification\devcenter\data-plane\Microsoft.DevCenter/preview/2022-03-01-preview/devcenter.json
```

Comment:
@chrissmiller Please use the links from the [spec repo](https://github.com/Azure/azure-rest-api-specs), so others could re-use this readme to generate codes.

By the way does the relevant swagger file merge into spec main branch?

Reply:
Thanks, fixed! The APIs are in spec main -- I am linking to a commit under PR to update the operation groups based on arch board feedback, but that's just to ensure operation names are consistent with other SDKs